applications/crossbar/doc/ref/about.md

```diff
@@ -8,11 +8,11 @@
 
 #### Fetch
 
-> GET /v2/accounts/{ACCOUNT_ID}/about
+> GET /v2/about
 
 ```curl
 curl -v -X GET \
     -H "X-Auth-Token: {AUTH_TOKEN}" \
-    http://{SERVER}:8000/v2/accounts/{ACCOUNT_ID}/about
+    http://{SERVER}:8000/v2/about
```

Comment:
This is a lie. Without accountid it returns 403. Please revert the "about" change you made all sneakily.
Please make multiple small PRs for unrelated things.